doc/TESTING.md

```diff
@@ -50,8 +72,12 @@ To uninstall Nicotine+, run:
 pip uninstall nicotine-plus
 ```
 
+## Source
+
 ### Git
 
+Running Nicotine+ using Git requires a [MinGW environment](https://en.wikipedia.org/wiki/MinGW) on Windows and [Homebrew](https://brew.sh/) on macOS. Note that problems encountered using this installation method on OSes other than GNU/Linux variants are unlikely to receive support.
```

Comment:
Delete sentence: "Note that problems encountered using this installation method on OSes other than GNU/Linux variants are unlikely to receive support."

It is obvious that running anything from a local Git folder is unsupported.